docker-compose/monitor/docker-compose.yml

```diff
@@ -11,11 +11,7 @@ services:
       - METRICS_STORAGE_TYPE=prometheus
       - PROMETHEUS_SERVER_URL=http://prometheus:9090
     ports:
-      - "14250:14250"
-      - "14268:14268"
-      - "6831:6831/udp"
       - "16686:16686"
-      - "16685:16685"
```

Comment:
isn't one of these ports still needed to receive data from OTEL collector?

Reply:
No, all ports are accessible between containers. Only the ports that need to be exposed to the "outside world" should be listed under `ports`.